css/style.css

```diff
@@ -125,10 +125,11 @@ button.button-inline:hover {
 	width: 100%;
 	bottom: 0px;
 	top: 44px;
+	margin-top: 10px;
 }
 
-#board #innerBoard {
-	padding: 10px;
+#innerBoard {
+	padding: 0 10px 20px 10px;
```

Comment:
why complicate padding? You essentially just move top padding over to margin-top for `#board`.

Edit: in fact, you could remove the whole declaration by just giving `padding: 10px` to `#board` element